### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ArsenSkandalenko**
[DEMO LINK](https://ArsenSkandalenko.github.io/layout_catalog/)
[TEST REPORT LINK](https://ArsenSkandalenko.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -22,6 +22,389 @@
...
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+
+        <article class="card__info card__info--title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </article>
```
Comment (anastasiiavorobiova) [inline src/index.html:120]:
```suggestion
        <h2 class="card__info card__info--title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>
```

File: src/index.html [hunk 2]
```diff
@@ -22,6 +22,389 @@
...
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="catalog">
+      <div
```
Comment (anastasiiavorobiova) [inline src/index.html:108]:
```suggestion
      <article
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

File: src/index.html [hunk 3]
```diff
@@ -22,6 +22,389 @@
...
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+
+        <article class="card__info card__info--title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </article>
```
Comment (anastasiiavorobiova) [inline src/index.html:158]:
```suggestion
        <h2 class="card__info card__info--title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>
```

File: src/index.html [hunk 4]
```diff
@@ -22,6 +22,389 @@
...
+        </a>
+      </div>
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+
+        <article class="card__info card__info--title">
```
Comment (anastasiiavorobiova) [inline src/index.html:193]: Fix the same issue further in the code

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done

**Comment by ArsenSkandalenko** [comment]
> Great work! Consider leveraging semantic tags for the title

Я трохи не розумію що я маю виправити :(

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Pls don’t ask for help in github, because this way mentors are only able to see your questions once the review is requested. Instead, pls always ask for help in **fe_chat** if you don't understand something so mentors can quickly help you

File: src/index.html [hunk 5]
```diff
@@ -22,6 +22,404 @@
...
+            class="card__icon"
+            src="images/imac.jpeg"
+            alt="iMac"
+          />
+
+          <h2 class="card__productName">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <div class="card__productCode">Product code: 195434</div>
```
Comment (volodymyr-soltys97) [inline src/index.html:129]: Why do you use div for text? You need to use semantic tag, so pls fix it everywhere
```suggestion
          <p class="card__productCode">Product code: 195434</p>
```

File: src/index.html [hunk 6]
```diff
@@ -22,6 +22,404 @@
...
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__icon"
+            src="images/imac.jpeg"
+            alt="iMac"
+          />
+
+          <h2 class="card__productName">
```
Comment (volodymyr-soltys97) [inline src/index.html:125]: Fix class everywhere
```suggestion
          <h2 class="card__title">
```

File: src/index.html [hunk 7]
```diff
@@ -22,6 +22,404 @@
...
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-count">Reviews: 5</div>
+          </div>
+
+          <div class="card__price-container">
```
Comment (volodymyr-soltys97) [inline src/index.html:143]:
```suggestion
          <div class="card__price">
```

File: src/index.html [hunk 8]
```diff
@@ -22,6 +22,404 @@
...
+            <div class="card__price-label">Price:</div>
+            <div class="card__price">$2,199</div>
```
Comment (volodymyr-soltys97) [inline src/index.html:145]:
```suggestion
            <div class="card__price-value">$2,199</div>
```

File: src/index.html [hunk 9]
```diff
@@ -22,6 +22,404 @@
...
+
+          <a
+            class="card__buy"
+            href="#"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+        <div class="card">
```
Comment (volodymyr-soltys97) [inline src/index.html:156]: Add the blank line between the cards
```suggestion
        </div>
        
        <div class="card">
```

File: src/index.html [hunk 10]
```diff
@@ -22,6 +22,404 @@
...
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
```
Comment (volodymyr-soltys97) [inline src/index.html:113]:
```suggestion
    </header>
    
    <main class="main">
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, you need to fix the demo link, make a deploy again and check it after on the demo link 
image

**Pull request closed without merging**